pygmt/datasets/earth_relief.py

```diff
@@ -16,7 +16,23 @@
 
 @kwargs_to_strings(region="sequence")
 def load_earth_relief(
-    resolution="01d",
+    resolution: Literal[
+        "01d",
+        "30m",
+        "20m",
+        "15m",
+        "10m",
+        "06m",
+        "05m",
+        "04m",
+        "03m",
+        "02m",
+        "01m",
+        "30s",
+        "15s",
+        "03s",
+        "01s",
+    ] = "01d",
```

Comment:
Or we can write the long list in a single line and let `ruff` ignore it so that ruff won't reformat it.

Reply:
Hm. Looks a bit weird. I have a slight preference for writing it in one line. But leaving it as is should also be OK with me.